stages/org.osbuild.ovf.meta.json

```diff
@@ -18,6 +18,11 @@
           "description": "The vmdk image filename present in the root of the tree",
           "type": "string",
           "pattern": "[a-zA-Z0-9+_.-]+.vmdk"
+        },
+        "ostype": {
+          "description": "The kind of installed guest operating system",
+          "type": "string",
+          "default": "other26xLinux64Guest"
```

Comment:
How long is the list of valid values here?  Does it make sense to make it an enum to verify that only valid values are used?

Reply:
According to the list I found it may be quite long: https://docs.vmware.com/en/VMware-HCX/4.9/hcx-user-guide/GUID-D4FFCBD6-9FEC-44E5-9E26-1BD0A2A81389.html
Also we would need to track this list in case something will change.
This whole PR is related to QA request where they would like to be able to "eventually" overwrite default value so I guess in the normal circumstances this wount be used that frequent:)